tests/lib/nested.sh

```diff
@@ -148,6 +149,7 @@ nested_get_ubuntu_image_url_for_vm() {
         esac
 }
 
+# shellcheck disable=SC2120
```

Comment:
why does this need to be disabled? shouldn't shellcheck see the usages of this function from the task.yaml when spread-shellcheck runs? if not then it's fine to leave this in as-is, I may just be confused about how shellcheck is invoked on our git tree